docs/conf.py

```diff
@@ -200,6 +200,8 @@ def __dir__(self):
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
+html_extra_path = ['_static/screenshots']
```

Comment:
We need to copy the contents of this folder to the build directory. This is one way to do it but we could also do it in the Makefile.

NB sphinx will copy any image using the `.. image:` directive automatically but, here, the table is actually a raw html table so the copying doesn't happen.